lib/entity.js

```diff
@@ -105,12 +149,34 @@ Entity.prototype.replay = function replay (events) {
   this.replaying = false;
 };
 
+
+/**
+ * Create a snapshot of the current state of the entity instance.
+ *
+ * Here the instance's snapshotVersion property is set to the current version,
+ * then the instance is deep-cloned and the clone is trimmed of the internal
+ * sourced attributes using trimSnapshot and returned.
+ *
+ * [CONFIRM] - Looking at https://lodash.com/docs#cloneDeep, I couldn't see what
```

Comment:
I think this is simply that lodash has changed since this was implemented. Just want to confirm that I understood it correctly.